rdr_service/tools/tool_libs/retention_metrics.py

```diff
@@ -318,39 +450,72 @@ def run(self):
 def run():
     # Set global debug value and setup application logging.
     setup_logging(
-        _logger, tool_cmd, "--debug" in sys.argv, "{0}.log".format(tool_cmd) if "--log-file" in sys.argv else None
+        _logger,
+        tool_cmd,
+        "--debug" in sys.argv,
+        "{0}.log".format(tool_cmd) if "--log-file" in sys.argv else None,
     )
     setup_i18n()
 
     # Setup program arguments.
     parser = argparse.ArgumentParser(prog=tool_cmd, description=tool_desc)
-    parser.add_argument("--debug", help="enable debug output", default=False, action="store_true")  # noqa
-    parser.add_argument("--log-file", help="write output to a log file", default=False, action="store_true")  # noqa
-    parser.add_argument("--project", help="gcp project name", default="localhost")  # noqa
+    parser.add_argument(
+        "--debug", help="enable debug output", default=False, action="store_true"
+    )  # noqa
+    parser.add_argument(
+        "--log-file",
+        help="write output to a log file",
+        default=False,
+        action="store_true",
+    )  # noqa
+    parser.add_argument(
+        "--project", help="gcp project name", default="localhost"
+    )  # noqa
     parser.add_argument("--account", help="pmi-ops account", default=None)  # noqa
-    parser.add_argument("--service-account", help="gcp iam service account", default=None)  # noqa
-    subparser = parser.add_subparsers(title='action', dest='action',
-                                      help='action to perform, such as qc')
+    parser.add_argument(
+        "--service-account", help="gcp iam service account", default=None
+    )  # noqa
+    subparser = parser.add_subparsers(
+        title="action", dest="action", help="action to perform, such as qc"
+    )
 
     load_parser = subparser.add_parser("load")
-    load_parser.add_argument("--file-upload-time", help="bucket file upload time (as string) of metrics file", type=str)
+    load_parser.add_argument(
+        "--file-upload-time",
+        help="bucket file upload time (as string) of metrics file",
+        type=str,
+    )
 
     qc_parser = subparser.add_parser("qc")
     qc_parser.add_argument("--bucket-file", help="bucket_file_path", type=str)
     qc_parser.add_argument("--csv", help="local CSV file path", type=str)
 
     recalc_parser = subparser.add_parser("recalc")
-    recalc_parser.add_argument('--from-file', help='file of ids to recalculate', default='', type=str)  # noqa
-    recalc_parser.add_argument("--id", help="comma-separated list of ids to recalculate", type=str, default=None)
+    recalc_parser.add_argument(
+        "--from-file", help="file of ids to recalculate", default="", type=str
+    )  # noqa
+    recalc_parser.add_argument(
```

Comment:
Combining this with the `--recalc` command/action will work, but it might warrant its own action/parser and dedicated tool class (whose `run` method is essentially the `handle_mismatches` method), since it's not actually doing recalculations?